images/dockerregistry/Dockerfile

```diff
@@ -6,16 +6,22 @@
 #
 FROM openshift/origin-base
 
-COPY config.yml $REGISTRY_CONFIGURATION_PATH
-COPY bin/dockerregistry /dockerregistry
+RUN INSTALL_PKGS="origin-dockerregistry" && \
+    yum install -y ${INSTALL_PKGS} && \
+    rpm -V ${INSTALL_PKGS} && \
+    yum clean all
 
-LABEL io.k8s.display-name="OpenShift Origin Image Registry" \
-      io.k8s.description="This is a component of OpenShift Origin and exposes a Docker registry that is integrated with the cluster for authentication and management."
+COPY config.yml ${REGISTRY_CONFIGURATION_PATH}
+
+LABEL io.k8s.display-name="OpenShift Container Platform Image Registry" \
+      io.k8s.description="This is a component ofOpenShift Container Platform and exposes a Docker registry that is integrated with the cluster for authentication and management." \
+      io.openshift.tags="openshift,docker,registry"
 
 # The registry doesn't require a root user.
 USER 1001
 EXPOSE 5000
 VOLUME /registry
-ENV REGISTRY_CONFIGURATION_PATH=/config.yml
+ENV REGISTRY_CONFIGURATION_PATH=/config.yml \
+	DOCKER_REGISTRY_URL=${DOCKER_REGISTRY_SERVICE_HOST}:${DOCKER_REGISTRY_SERVICE_PORT}
 
-CMD DOCKER_REGISTRY_URL=${DOCKER_REGISTRY_SERVICE_HOST}:${DOCKER_REGISTRY_SERVICE_PORT} /dockerregistry ${REGISTRY_CONFIGURATION_PATH}
```

Comment:
@legionus can you help me understand why the registry works when this `DOCKER_REGISTRY_URL` env is not set? It seems like we use it only for testing.